docs/content/preview/migrate/known-issues.md

```diff
@@ -536,13 +952,100 @@ PRIMARY KEY (employee_id, hire_date)) PARTITION BY RANGE (hire_date) ;
 An example schema on the source database is as follows:
 
 ```sql
-create index ON timestamp_demo (ts);
+CREATE INDEX ON timestamp_demo (ts);
 ```
 
-Suggested change to the schema is to add the `asc` clause as follows:
+Suggested change to the schema is to add the `ASC` clause as follows:
 
 ```sql
-create index ON timestamp_demo (ts asc);
+CREATE INDEX ON timestamp_demo (ts ASC);
 ```
 
 ---
+
+#### Exporting data with names for tables/functions/procedures using special characters fails
+
+**GitHub links**: [Issue #636](https://github.com/yugabyte/yb-voyager/issues/636), [Issue #688](https://github.com/yugabyte/yb-voyager/issues/688), [Issue #702](https://github.com/yugabyte/yb-voyager/issues/702)
+
+**Description**: If you define complex names for your source database tables/functions/procedures using backticks or double quotes for example, \`abc xyz\` , \`abc@xyz\`, or "abc@123", the migration hangs during the export data step.
+
+**Workaround**: Rename the objects (Tables/functions/procedures) on the source database to something without special characters.
```

Comment:
Can we mention `special characters/whitespaces` as whitespaces are a more common use case. Thanks.